Bottom: ae4d2ecbaffb2dbf309c1021868c7733914a2a61
Top:    c4ff4ba71e24fbd39659cf6d0163ca44b805b898
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-06-05 12:07:55 +0100

Disable texturing the board's soldermask


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 550653b..10f11c7 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1469,6 +1469,7 @@ GhidDrawMask (int side, BoxType * screen)
   gui->graphics->set_color (out->fgGC, PCB->MaskColor);
   ghid_set_alpha_mult (out->fgGC, thin ? 0.35 : 1.0);
 
+#if 0
   if (first_run) {
     glGenTextures (1, &texture);
     glBindTexture (GL_TEXTURE_2D, texture);
@@ -1496,6 +1497,7 @@ GhidDrawMask (int side, BoxType * screen)
   glTexParameterf (GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_REPEAT);
   glTexParameterf (GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_REPEAT);
   glEnable (GL_TEXTURE_2D);
+#endif
 
   if (!PCB->Data->outline_valid) {
 
@@ -1517,10 +1519,12 @@ GhidDrawMask (int side, BoxType * screen)
   poly_FreeContours (&polygon.NoHoles);
   ghid_set_alpha_mult (out->fgGC, 1.0);
   hidgl_flush_triangles (&buffer);
+#if 0
   glDisable (GL_TEXTURE_GEN_S);
   glDisable (GL_TEXTURE_GEN_T);
   glBindTexture (GL_TEXTURE_2D, 0);
   glDisable (GL_TEXTURE_2D);
+#endif
   hidgl_shader_activate (circular_program);
 
   gui->graphics->use_mask (HID_MASK_OFF);
